src/feedCLAIM/index.tsx

```diff
@@ -0,0 +1,131 @@
+          </Typography>
+          <Typography sx={{ padding: '10px 0 0 10px' }}>claim: 'rated',</Typography>
+          <Typography sx={{ padding: '10px 0 0 10px' }}>
+            object: 'http://trustclaims.whatscookin.us/local/company/VEJA',
+          </Typography>
+        </CardContent>
+        <CardActions disableSpacing>
+          <ExpandMore expand={expanded} onClick={handleExpandClick} aria-expanded={expanded} aria-label='show more'>
+            <ExpandMoreIcon />
+          </ExpandMore>
+        </CardActions>
+        <Collapse in={expanded} timeout='auto' unmountOnExit>
+          <CardContent>
+            <Typography sx={{ padding: '10px 0 0 10px' }}>
+              Statement: 'The VEJA project creates a positive chain. Sneakers are made differently using organic,
+              agroecological and fairtrade cotton to make the sneakers canvas, wild rubber from the Amazonian Forest for
+              its soles and innovative materials such as recycled plastic bottles to create a new mesh. VEJA is about
+              minimalism and innovation. Their logistics and shipping run by Ateliers Sans Frontières, a rehabilitation
+              association. Made in Brazil.', ,
+            </Typography>
+            <Typography sx={{ padding: '10px 0 0 10px' }}>Aspect: 'social:impact'</Typography>
+            <Typography sx={{ padding: '10px 0 0 10px' }}>
+              SourceURI: 'https://data.world/blab/b-corp-impact-data'
+            </Typography>
+            <Typography sx={{ padding: '10px 0 0 10px' }}>EffectiveDate: null</Typography>
+            <Typography sx={{ padding: '10px 0 0 10px' }}> Confidence: 0.8 </Typography>
+          </CardContent>
+        </Collapse>
+      </Card>
+
+      <Card sx={{ maxWidth: 'fit', m: '20px', borderRadius: '4px', border: '2px solid #80B8BD' }}>
+        <CardMedia component='img' height='194' src={polygon3} />
+        <CardContent>
+          <Typography sx={{ top: '10px', padding: '10px 0 0 10px' }}>
+            subject: 'https://www.bcorporation.net/',
+          </Typography>
+          <Typography sx={{ padding: '10px 0 0 10px' }}>claim: 'rated',</Typography>
+          <Typography sx={{ padding: '10px 0 0 10px' }}>
+            object: 'http://trustclaims.whatscookin.us/local/company/VEJA',
+          </Typography>
+        </CardContent>
+        <CardActions disableSpacing>
+          <ExpandMore expand={expanded} onClick={handleExpandClick} aria-expanded={expanded} aria-label='show more'>
+            <ExpandMoreIcon />
+          </ExpandMore>
+        </CardActions>
+        <Collapse in={expanded} timeout='auto' unmountOnExit>
+          <CardContent>
+            <Typography sx={{ padding: '10px 0 0 10px' }}>
+              Statement: 'The VEJA project creates a positive chain. Sneakers are made differently using organic,
+              agroecological and fairtrade cotton to make the sneakers canvas, wild rubber from the Amazonian Forest for
```

Comment:
in future remember to make it dynamic, there should never be a merge to master with any hardcoded text except in tests

Reply:
okay, i understand ,I removed all of that hard code and make the card empty, and the next task , will be dynamic, like our plan